Sources/ShopifyCheckoutSheetKit/CheckoutBridge.swift

```diff
@@ -110,8 +110,17 @@ extension CheckoutBridge {
 					self = .checkoutComplete(event: emptyCheckoutCompletedEvent)
 				}
 			case "error":
-				// needs to support .checkoutUnavailable by parsing error payload on body
-				self = .checkoutExpired
+				let errorDecoder = CheckoutErrorEventDecoder()
+				let error = errorDecoder.decode(from: container, using: decoder)
+
+				switch error.group {
+				case .unrecoverable:
+					self = .checkoutUnavailable(message: error.reason)
+				case .expired:
```

Comment:
Is expired a new group that's being added?

Reply:
Yeah, we've added this in checkout